src/components/layout/layout.tsx

```diff
@@ -82,6 +81,7 @@ interface LayoutProps {
   translation?: string;
   leadbox?: LeadboxProps;
   breadcrumb?: BreadcrumbEntry[];
+  mainAs?: any;
```

Comment:
We should not overwrite `main`.

Here is how it's implemented in zeit.de:
<img width="881" alt="grafik" src="https://user-images.githubusercontent.com/7814926/168765562-53577fb9-7853-4cda-93f0-206c49f22d5c.png">

So `main` should always stay there. A solution could be to add an additional HTML tag inside main, that is either a `div` or an `article`.

Reply:
okay, yes that seems to be convenient. I added such a wrapper. Please have another look